src/main/resources/infernal-expansion.mixins.json

```diff
@@ -21,7 +21,8 @@
     "common.MixinShearsItem",
     "common.MixinShovelItem",
     "common.MixinStriderEntity",
-    "common.WorldCarverAccessor"
+    "common.WorldCarverAccessor",
+    "common.MixinIngredientAccessor"
```

Comment:
please make sure to list mixins alpabetically in this json file

Reply:
I didnt notice that was the case, ill change that